src/runtime_src/core/pcie/tools/xbmgmt/cmd_hotplug.cpp

```diff
@@ -29,9 +29,11 @@
 #define SYSFS_PATH      "/sys/bus/pci/devices"
 #define XILINX_VENDOR   "0x10ee"
 #define XILINX_US       "0x9134"
+#define POLL_TIMEOUT    60      /* Set a pool timeout as 60sec */
 
 static int hotplugRescan(void);
 static int removeDevice(const std::shared_ptr<pcidev::pci_device> dev);
+static int shutdownDevice(const std::shared_ptr<pcidev::pci_device> dev);
```

Comment:
change to 'const pcidev::pci_device* dev' and call as 'shutdownDevice(uDev.get())'
Ditto for removeDevice.

Reply:
I have updated the code. But it is a bit different way. Passed device index as a parameter. The same implementation is followed in other files. Please review it once.